src/NUnitFramework/framework/Internal/Tests/TestAssembly.cs

```diff
@@ -39,25 +39,25 @@ public class TestAssembly : TestSuite
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="TestAssembly"/> class
-        /// specifying the Assembly and the path from which it was loaded.
+        /// specifying the Assembly and the name of the assembly.
         /// </summary>
         /// <param name="assembly">The assembly this test represents.</param>
-        /// <param name="path">The path used to load the assembly.</param>
-        public TestAssembly(Assembly assembly, string path)
-            : base(path)
+        /// <param name="name">The name of the assembly.</param>
```

Comment:
To be more direct about the fact that this parameter just controls a display name, could we say, "The desired name for the test suite?"